utils/m0reportbug

```diff
@@ -105,14 +105,15 @@ main() {
                 warn "$IN_DIR: no such directory"
                 continue
             }
-            [[ -L $IN_DIR ]] && {
+            # This check is commented because R2 does not have symlinks for /var/motr unlike R1.
```

Comment:
Is it causing any issue ?

Reply:
Yes, if /var/motr is a symlink to any other path then m0reportbug would not pack the m0trace files populated on that path.

Reply:
Do we see any case where /var/motr is sym link and pointing to some other path in R2 other than dev setup? 
In R2 I think we will always be having /var/motr as directory  only.